microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -0,0 +1,96 @@
+
+In general, the website's internationalization user experiences are only as good as the regional code that they're able to determine for the current user. 
+
+A regional code is a two-letter language code followed by a hyphen and a two-letter region code.  Microsoft Edge uses ICU, which is an implementation of the Unicode standard's Internationalization API.  See [ICU](https://icu.unicode.org/home).
+
+To set the user's language preferences, Microsoft Edge gets the default values from the OS when Microsoft Edge is first installed. The user can then change the language preferences in `edge://settings/languages` and sort them in priority order.
+
+| Data Source | Result |
+|--------|-------------------------|
+| **Preferred language** in the **setting** page is the language selected for rendering Microsoft Edge. | This language/region is also used for formatting values such as date, time, and numbers. |
+|  `accept-language` header | This is used for Website Server-side processing. Microsoft Edge sends the language list (as regional codes) specified in **edge://settings/languages**. | 
+| `navigator.language` | This is the preferred language (at the top of the list) used to render websites. |
+
+## The Sharing setting
+Customers have three settings on Microsoft Edge to select the level of information their browser will share about their regions:
+
+* Never 
+* Limited (default) 
+* Always (not recommended). 
+
+#### The Never option
+To minimize the risks, users can disable the sharing of OS regional information by selecting **Never**. The **Never** option still allows websites to read regional information, but does block any specific OS regional customizations that the user has made from being exposed to website content. Instead, the browser's display language is used to derive a default region. 
+
+#### The Limited option (default)
+The default recommended value is **limited**. With this setting, the above risks are mitigated by ensuring that mismatched combinations of language and region aren't revealed to websites. The **limited** value ensures that only valid combinations of the user's current browser display language preference and a given region value are used. **Limited** ensures that websites won't see a regional code anomaly (though it could still look unusual per geographic inference), and also ensures that for language processing, websites will at least provide a consistent language experience (not mixing languages in weird combinations). The **limited** setting compares the language part of the regional code that is derived from the browser’s display language, and the regional part coming from the OS setting, and if the language parts match and if the resulting regional code is a valid combination, will pass the result through to the website. If the language in the message doesn't match, the feature acts as if **never** was selected (the regional information falls back to being derived from the browser’s display language). 
```

Comment:
```suggestion
The default, recommended value is **Limited**. When using the **Limited** setting, the above risks are mitigated by ensuring that mismatched combinations of language and region aren't revealed to websites. The **Limited** option ensures that only valid combinations of the user's current browser display language preference and a given region value are used. **Limited** ensures that websites won't see a regional code anomaly (though it could still look unusual per geographic inference), and also ensures that for language processing, websites will at least provide a consistent language experience (not mixing languages in weird combinations). The **Limited** option compares the language part of the regional code that is derived from the browser’s display language, and the regional part coming from the OS setting, and if the language parts match and if the resulting regional code is a valid combination, will pass the result through to the website. If the language in the message doesn't match, the feature acts as if **Never** was selected (the regional information falls back to being derived from the browser’s display language). 
```
also break up paragraph, and reduce use of parentheses

Global when referring to UI: capitalize per UI